packages/quantic/force-app/main/default/lwc/quanticHeadlessLoader/__tests__/headlessLoader.test.js

```diff
@@ -6,7 +6,7 @@ import {
   getHeadlessEngine,
```

Comment:
I was not able to run the tests locally. Any thoughts on these errors?

```
> @coveo/quantic@0.22.0 test:unit
> lwc-jest

error Invalid sourceApiVersion found in sfdx-project.json. Expected 52.0, found 50.0
error TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received an instance of Object
```

Reply:
I think the installed version of [sfdx-lwc-jest](https://www.npmjs.com/package/@salesforce/sfdx-lwc-jest) is expecting api version 52. Could you try bumping the apiVersion in sfdx-project.json as well as the  js-meta.xml files to 52.0?

Reply:
Changing the version to 52 solves the first error, but not the second 🤷 I wonder if there is a windows incompatability since the tests are running on jenkins/linux.

Reply:
hmm i get that error as well and i'm on mac. that's odd though. last time we made a contribution the tests had no issues. Maybe the result of a change in the dependencies?

Reply:
I recall merging some PRs opened by dependabot for quantic dependencies. Still though, wouldn't we expect the tests to also fail on jenkins? There must be something wrong with our local environments, hmm